presentation/src/main/java/com/dcs/presentation/ui/trend/TrendRoute.kt

```diff
@@ -194,7 +196,7 @@ fun CustomScrollableTabRow(
 }
 
 @Composable
-fun TrendMovieUiStateScreen(
+fun TrendMovieUiState(
```

Comment:
```suggestion
fun TrendMovies(
```
로 바꾸고 64줄에 TrendMovies -> TrendMovieContens 변경하는게 좋지 않을까요? 가급적이면 함수에서 UiState를 사용 안하는게 좋다고 생각해요. 전달되는 변수명이랑 혼동을 줘서요.

Reply:
TrendMovieContents -> TrendMovie -> MovieItem 순으로 수정 [bae8062](https://github.com/Edgar-J-Shin/AllAboutMovie/pull/29/commits/bae8062b1490a5c82d49cee0d5cbcb9e1ba46da5)

Reply:
TrendMovies -> TrendMovieContents -> TrendMovie -> MovieItem 순인거죠?

Reply:
ㅇㅋ 재수정 완료 [31199c8](https://github.com/Edgar-J-Shin/AllAboutMovie/pull/29/commits/31199c8e6fb0b2b8ebe9885e4681ad0cb09352ab)